src/cards/util.ts

```diff
@@ -6,6 +6,7 @@ export interface Modifier {
     quantity: number;
     // If true, the modifier will not be removed from the unit on death 
     keepOnDeath?: boolean;
+    keepBetweenLevels?: boolean;
```

Comment:
removeModifier already prevents removing runes.  Is `keepBetweenLevels` necessary?
```  
if (modifier && modifier.costPerUpgrade) {
    // Modifier is a Rune and should NOT be removed
    return;
  }```

Reply:
I think I know why you added this back, see my comment in Unit.ts for more rationale for removing it

Reply:
Here was the thought process:
- Want to add a Bounty System: One random unit is given a bounty, and killing that unit grants special rewards/effects. It creates a new assassin-like playstyle with unique challenges, strategies, etc.
- Players can engage with the bounty system via runes that have bounty related effects, such as restoring health/stamina/mana when they claim a bounty
- Bounties should only begin appearing if the player has at least one of these runes, as bounties have no innate purpose/effect without them
- The system needs some kind of handler for spawning bounties. Putting it in each individual modifier is not maintainable, and putting it in Underworld turn logic feels invasive/incorrect
- Decided to write this system through a modifier called Bounty Hunter:
  - The Bounty Hunter modifier is what controls the bounty system by spawning bounties and highlighting them through onDrawSelected and stuff like that
  - Any bounty related rune will also give the player a persistent Bounty Hunter modifier to effectively "enable" the bounty system
  - Bounty Hunter needs to be persistent through levels, like the runes it comes with, but it should not share any other features of runes (has no effect on its own, isn't purchasable)

Reply:
Hmm, I see, okay I'm on board

Reply:
Do you forsee this changing over the lifetime of the game for a modifier instance, because if not I think we should put it in the modifier source rather than the modifier instance state. This will allow us to just check for it in `removeModifier` (in the same place that we check `costPerUpgrade`

Reply:
I'm not 100% sure what you mean by forsee it changing. I don't think bounty hunter would ever _not_ persist through death, if that's what you're asking. I'll move it to the `Modifiers` interface with `costPerUpgrade`